cmd/kyma/alpha/deploy/localdomain.go

```diff
@@ -0,0 +1,72 @@
+package deploy
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+)
+
+// LocalKymaDevDomain contains the default domain name if Kyma is locally deployed
+const LocalKymaDevDomain = "local.kyma.dev"
+
+// KymaLocalDomain configures the local kyma domain in Kubernetes DNS
+type KymaLocalDomain struct {
+	coreDNSConfig []byte
+	kubeClient    kubernetes.Interface
+}
```

Comment:
In my opinion, the name `KymaLocalDomain` does not really reflect what this type is doing (and is too similar to the constant `LocalKymaDevDomain` above). Maybe something like `LocalDomainDNSConfigurer`?

Reply:
Agree, the name is not self-expressive enough. I adjusted it to "DNSConfigurer".